tests/adapters/request_network/test_adapter.py

```diff
@@ -9,59 +9,16 @@
 
 
 def describe_adapter() -> None:
-    def it_validate_rn_invoice_api_url() -> None:
+    def it_validates_rn_invoice_api_url() -> None:
         with pytest.raises(ValueError):
             adapter.RequestNetworkInvoiceAdapter(request_network_invoice_api_url="")
 
-    def it_validate_rn_subgraph_endpoint_url() -> None:
+    def it_validates_rn_subgraph_endpoint_url() -> None:
         with pytest.raises(ValueError):
             adapter.RequestNetworkInvoiceAdapter(
                 request_network_subgraph_endpoint_url=""
             )
 
-    def it_validate_chain() -> None:
-        with pytest.raises(ValueError):
-            adapter.RequestNetworkInvoiceAdapter(chain=None)
-
-    def describe_get_payments() -> None:
```

Comment:
Removed the test because I'm going to have this adapter depend on `request_client` instead, and the test is covered there.